libdnf/repo/repo_downloader.hpp

```diff
@@ -141,8 +115,7 @@ class RepoDownloader {
     std::vector<std::pair<std::string, std::string>> metadata_locations;
     std::map<std::string, std::string> metadata_paths;
 
-    std::unique_ptr<LrHandle> handle;
-    std::unique_ptr<LrResult> lr_result;
+    std::unique_ptr<LibrepoHandle> handle;
```

Comment:
Since this is now our class I'd say it would be a bit better wrapping it in `std::optional`, it no longer needs to be dynamically allocated AFAICS?